cmd/bfgd/bfgd.go

```diff
@@ -27,119 +22,28 @@ const (
-			Help:         "bitcoin start height that serves as genesis",
-			Print:        config.PrintAll,
-			Required:     true,
-		},
-		"BFG_LOG_LEVEL": config.Config{
-			Value:        &cfg.LogLevel,
-			DefaultValue: defaultLogLevel,
-			Help:         "loglevel for various packages; INFO, DEBUG and TRACE",
-			Print:        config.PrintAll,
-		},
-		"BFG_POSTGRES_URI": config.Config{
-			Value:        &cfg.PgURI,
-			DefaultValue: "",
-			Help:         "postgres connection URI",
-			Print:        config.PrintSecret,
-			Required:     true,
-		},
-		"BFG_PUBLIC_ADDRESS": config.Config{
-			Value:        &cfg.PublicListenAddress,
-			DefaultValue: bfgapi.DefaultPublicListen,
-			Help:         "address and port bfgd listens on for public, authenticated, websocket connections",
-			Print:        config.PrintAll,
-		},
-		"BFG_PRIVATE_ADDRESS": config.Config{
-			Value:        &cfg.PrivateListenAddress,
-			DefaultValue: bfgapi.DefaultPrivateListen,
-			Help:         "address and port bfgd listens on for private, unauthenticated, websocket connections",
-			Print:        config.PrintAll,
-		},
-		"BFG_PROMETHEUS_ADDRESS": config.Config{
-			Value:        &cfg.PrometheusListenAddress,
-			DefaultValue: "",
-			Help:         "address and port bfgd prometheus listens on",
-			Print:        config.PrintAll,
-		},
-		"BFG_PPROF_ADDRESS": config.Config{
-			Value:        &cfg.PprofListenAddress,
-			DefaultValue: "",
-			Help:         "address and port bfgd pprof listens on (open <address>/debug/pprof to see available profiles)",
-			Print:        config.PrintAll,
-		},
-		"BFG_REQUEST_LIMIT": config.Config{
-			Value:        &cfg.RequestLimit,
-			DefaultValue: bfgapi.DefaultRequestLimit,
-			Help:         "maximum request queue depth",
-			Print:        config.PrintAll,
-		},
-		"BFG_REQUEST_TIMEOUT": config.Config{
-			Value:        &cfg.RequestTimeout,
-			DefaultValue: bfgapi.DefaultRequestTimeout,
-			Help:         "request timeout in seconds",
-			Print:        config.PrintAll,
-		},
-		"BFG_TRUSTED_PROXIES": config.Config{
-			Value:        &cfg.TrustedProxies,
-			DefaultValue: []string{},
-			Help:         "trusted proxies IP addresses or CIDRs",
-			Print:        config.PrintAll,
-		},
-		"BFG_REMOTE_IP_HEADERS": config.Config{
-			Value:        &cfg.RemoteIPHeaders,
-			DefaultValue: []string{},
-			Help:         "list of headers used to obtain the client IP address (requires trusted proxies)",
-			Print:        config.PrintAll,
-		},
-		"BFG_BFG_URL": config.Config{
-			Value:        &cfg.BFGURL,
-			DefaultValue: "",
-			Help:         "public websocket address of another BFG you'd like to receive L2Keystones from",
-			Print:        config.PrintAll,
-		},
-		"BFG_BTC_PRIVKEY": config.Config{
-			Value:        &cfg.BTCPrivateKey,
-			DefaultValue: "",
-			Help:         "a btc private key, this is only needed when connecting to another BFG",
-			Print:        config.PrintSecret,
-		},
+		// Other configuration options here...
```

Comment:
The other configuration options must be defined here, please do not remove them.